packages/store/test/snapshots/test-transform.js.md

```diff
@@ -0,0 +1,363 @@
+# Snapshot report for `test/test-transform.js`
+
+The actual snapshot is saved in `test-transform.js.snap`.
+
+Generated by [AVA](https://avajs.dev).
```

Comment:
I followed the link and get the basic idea, but have questions: This is a generated file. Should we be checking it into git? Should I review the contents of the snapshot file? What question am I asking when I review a snapshot file?